include/alpaka/mem/global/DeviceGlobalCpu.hpp

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2024 Aurora Perego
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include "alpaka/mem/buf/cpu/Copy.hpp"
+#include "alpaka/mem/view/ViewPlainPtr.hpp"
+
+#include <type_traits>
+
+// memcpy specialization for device global variables
+namespace alpaka
+{
+    template<typename TViewSrc, typename TViewDstFwd, typename TQueue>
+    ALPAKA_FN_HOST auto memcpy(TQueue& queue, alpaka::DevGlobal<TViewDstFwd>& viewDst, TViewSrc const& viewSrc) -> void
+    {
+        typedef std::remove_all_extents_t<TViewDstFwd> T;
```

Comment:
I would prefer to use
```suggestion
        using T = std::remove_all_extents_t<TViewDstFwd>;
```
everywhere, instead of `typedef`.

And actually `Type` would make more sense than `T`, which is normally used for template arguments.